code/game/objects/structures/traps.dm

```diff
@@ -19,56 +33,43 @@
 		trap_effect(L)
 		animate(src, alpha = initial(alpha), time = time_between_triggers)
 
-/obj/structure/trap/examine(mob/user)
```

Comment:
It doesn't benefit anyone to detect a trap, since it is more visible like it's just been set off, but still armed.